src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -1662,6 +1684,32 @@ public override IOperation VisitCaughtException(ICaughtExceptionOperation operat
             throw ExceptionUtilities.Unreachable;
         }
 
+        private IOperation VisitNoneOperationExpression(IOperation operation)
+        {
+            int startingStackSize = _evalStack.Count;
+            foreach (IOperation child in operation.Children)
+            {
+                _evalStack.Push(Visit(child));
+            }
+
+            int numChildren = _evalStack.Count - startingStackSize;
```

Comment:
Are there any tests where `numChildren != operation.Children.Length`?

Reply:
Talked offline. This can't happen, I'll add assert to verify that. We aren't just using operation.Children.Length because Children is an IEnumerable.